### ISSUE TITLE
Issue with Spectral Norm at Nowcasting Model

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/satflow#73]
Probably a very stupid issue, but there seems to be a problem when wrapping torch layers with the `spectral_norm` in that PyTorch complains of in-place operations and crashes the training. Spectral normalization seems quite important for the model, so definitely need to get it working.

**Comment by jacobbieker** [comment]
https://github.com/openclimatefix/satflow/commit/2a2cfe6e77d764922cdd8928025c69094299ffef removes the spectral norm and now the model trains to some extant. Still need to get the spectral norm back into the model.
